aten/src/ATen/core/boxing/KernelFunction_impl.h

```diff
@@ -36,21 +36,16 @@ inline void KernelFunction::callBoxed(const OperatorHandle& opHandle, Stack* sta
     (*boxed_kernel_func_)(functor_.get(), opHandle, stack);
 }
 
-template<class Return, class... Args>
-inline Return callUnboxedKernelFunction(void* unboxed_kernel_func, OperatorKernel* functor, Args&&... args) {
```

Comment:
a code base searched showed the below call site as the only location where this function is called, so I thought it safe to inline.